lldb/test/API/tools/lldb-server/thread-name/main.cpp

```diff
@@ -1,4 +1,7 @@
 #include <pthread.h>
+#ifdef __OpenBSD__
```

Comment:
For consistency, can you change this to `if defined(__OpenBSD__)`? Not a big deal though

Reply:
Sure.

Reply:
Done.